common/djangoapps/enrollment/views.py

```diff
@@ -490,7 +491,7 @@ def get(self, request):
             return Response(enrollment_data)
         filtered_data = []
         for enrollment in enrollment_data:
-            course_key = CourseKey.from_string(enrollment["course_details"]["course_id"])
+            course_key = course_key_from_string_or_404(enrollment["course_details"]["course_id"])
```

Comment:
We don't need this here because we can assume that `api.get_enrollments` in L479 is only returning valid course ids.